streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -46,26 +55,27 @@ public void configure() throws IOException, Exception {
 	@VisibleForTesting
 	FhirConverter createFhirConverter(CamelContext camelContext) throws IOException, Exception {
 		FhirContext fhirContext = FhirContext.forDstu3();
-		String fhirBaseUrl = System.getProperty("openmrs.serverUrl") + System.getProperty("openmrs.fhirBaseEndpoint");
-		OpenmrsUtil openmrsUtil = new OpenmrsUtil(fhirBaseUrl, System.getProperty("openmrs.username"),
-		        System.getProperty("openmrs.password"), fhirContext);
-		FhirStoreUtil fhirStoreUtil = FhirStoreUtil.createFhirStoreUtil(System.getProperty("fhir.sinkPath"),
-		    System.getProperty("fhir.sinkUsername"), System.getProperty("fhir.sinkPassword"),
-		    fhirContext.getRestfulClientFactory());
+		DebeziumArgs params = DebeziumArgs.getInstance();
+		JCommander.newBuilder().addObject(params).build().parse(args);
+		String fhirBaseUrl = params.openmrsServerUrl + params.openmrsfhirBaseEndpoint;
+		OpenmrsUtil openmrsUtil = new OpenmrsUtil(fhirBaseUrl, params.openmrUserName, params.openmrsPassword, fhirContext);
+		FhirStoreUtil fhirStoreUtil = FhirStoreUtil.createFhirStoreUtil(params.fhirSinkPath, params.sinkUser,
+		    params.sinkPassword, fhirContext.getRestfulClientFactory());
 		ParquetUtil parquetUtil = new ParquetUtil(fhirContext);
 		camelContext.addService(new ParquetService(parquetUtil), true);
-		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil);
+		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumEventConfigPath);
 	}
 	
 	private String getDebeziumConfig() {
-		return "debezium-mysql:{{database.hostname}}?" + "databaseHostname={{database.hostname}}"
-		        + "&databaseServerId={{database.serverId}}" + "&databasePort={{database.port}}"
-		        + "&databaseUser={{database.user}}" + "&databasePassword={{database.password}}"
+		DebeziumArgs params = DebeziumArgs.getInstance();
```

Comment:
ditto here; just use `params` instance variable, initially once in the constructor.

Reply:
sure